ci_build_images/sles.Dockerfile

```diff
@@ -63,16 +63,14 @@ RUN zypper -n update \
     subversion \
     systemd-devel \
     wget \
-    # temporary add opensuse oss repo for some deps \
-    && zypper addrepo https://download.opensuse.org/distribution/leap/RELEASEVER/repo/oss/ repo-oss \
-    && sed -i "s/RELEASEVER/\$releasever/" /etc/zypp/repos.d/repo-oss.repo \
-    # temp add since repo (no 15.6 version) for eigen3 \
-    && zypper addrepo https://download.opensuse.org/repositories/science/SLE_15_SP5/science.repo \
-    && zypper --gpg-auto-import-keys ref science repo-oss \
+    # Add eigen3-devel and judy-devel from opensuse OSS repository.
```

Comment:
eigen3 was dropped as a dependency - MDEV-34942 so this can be removed.

libJudy1 is still 1.0.5 in the bci of 15 SP 7  so just note we're pulling the build dependy of judy-devel and this isn't a repo that we're now dependant on for release.